README.md

```diff
@@ -1,3 +1,4 @@
+#
```

Comment:
is this intentional?

Reply:
Yes, it is not valid markdown to start with ##, and this doesn't really change anything visually.

Reply:
interesting, didn't know!